packages/flutter_driver/lib/src/common/find.dart

```diff
@@ -252,3 +252,44 @@ class ByType extends SerializableFinder {
     return new ByType(json['type']);
   }
 }
+
+/// A Flutter driver command that retrieves a semantics id using a specified finder.
+/// 
+/// If the object returned by the finder does not have its own semantics node,
+/// then the semantics node of the first ancestor is returned instead.
+/// 
+/// If the finder returns multiple objects, or there are no semantics nodes then
+/// a [StateError] is thrown by the extension.
```

Comment:
Do we define anywhere what "the extension" is? Or is it just an implementation detail that the error is thrown by the extension and we should maybe omit that detail from documentation?

Reply:
An error thrown on the app side becomes an error thrown in the driver script, it isn't necessary to mention that is thrown on the device so i will omit this part